src/tools/Makefile.am

```diff
@@ -81,7 +81,7 @@ openpgp_tool_SOURCES = openpgp-tool.c util.c
 openpgp_tool_LDADD = $(OPTIONAL_OPENSSL_LIBS)
 iasecc_tool_SOURCES = iasecc-tool.c util.c
 iasecc_tool_LDADD = $(OPTIONAL_OPENSSL_LIBS)
-sc_hsm_tool_SOURCES = sc-hsm-tool.c util.c
+sc_hsm_tool_SOURCES = sc-hsm-tool.c util.c fread_to_eof.c
```

Comment:
I think this also needs to be added in Makefile.mak

Reply:
i attempted a change, but i'm not sure how to interpret this error

```
light.exe : error LGHT0103 : The system cannot find the file '..\src\tools\cardos-tool.exe' with type ''.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\WiX Toolset v3.11\\bin\light.exe"' : return code '0x67'
Stop.
Command exited with code 2
```

Reply:
```
   Creating library pkcs11-spy.lib and object pkcs11-spy.exp
	if EXIST pkcs11-spy.dll.manifest mt -manifest pkcs11-spy.dll.manifest -outputresource:pkcs11-spy.dll;2
Makefile.mak(62) : fatal error U1001: syntax error : illegal character '^' in macro
Stop.
```

Reply:
oops thanks, that seems to have fixed it!